APIs e2e testing using HTTP files and Rest Client (#1)
ISE-Neutrino/api-testing: README.md

# APIs Testing

> This repository demonstrates by example how to automate the APIs End-to-End(e2e) testing using Http files and Rest Client.

Reviewer: ```suggestion
This repository shows how to use Http files and Rest Client to automate the end-to-end (e2e) testing of APIs.
```
